app/CommandHandling/CommandBusServiceProvider.php

```diff
@@ -250,6 +253,15 @@ function () use ($container) {
             }
         );
 
+        $container->addShared(
+            MailerCommandBus::class,
+            function () {
+                $commandBus = $this->createResqueCommandBus('event_export', $this->container);
```

Comment:
Shouldn't this have it's own specific queue iso reusing the `event_export` queue?

Reply:
That part was a WIP, it has completely changed by now.